python/mlc_chat/cli/compile.py

```diff
@@ -101,6 +101,12 @@ def _check_system_lib_prefix(prefix: str) -> str:
         default="",
         help=HELP["overrides"] + ' (default: "%(default)s")',
     )
+    parser.add_argument(
+        "--debug-dump-path",
```

Comment:
nit: keep it concise

```suggestion
        "--debug-dump",
```